config/samples/flows_v1alpha1_flowcollector.yaml

```diff
@@ -47,14 +47,20 @@ spec:
         certFile: user.crt
         certKey: user.key
   loki:
-    url: 'http://loki:3100/'
+    url: 'http://loki.network-observability.svc:3100/'
```

Comment:
did you need this change / does it makes anything easier?
The problem I see by inferring namespace is that if you change the `spec.namespace`, then this url is likely invalid.
By sticking with just `http://loki` we assume loki is deployed next to the other components

Reply:
Yes, the certificate generated by Openshift is only valid for the full name.

Reply:
Ah I see, ok then